mode/examples/Demos/Graphics/BoxClock/BoxClock.pyde

```diff
@@ -5,21 +5,18 @@
     (C) Ben Alkov, 2014, licensed as APL 2.0 as part of processing.py
     (https://github.com/jdf/processing.py).
 '''
-# Creating a **filled** wireframe cube is non-obvious.
-# We need an opaque black cube...
-fillCube = None
-# ...and a transparent colored wireframe.
-edgeCube = PShape()
 
 
 def setup():
+    global fillCube
```

Comment:
Just to make sure I'm getting this:
`fillCube` is global (used **only** in `setup` and `drawShape`), defined in `setup`, and declared global in `setup`.
`edgeCube` is global (used **only** in `makeEdgeCube` and `drawShape`), defined in `makeEdgeCube`, but should be **declared** global in `setup`?